client/css/style.css

```diff
@@ -1202,7 +1202,7 @@ kbd {
 }
 
 #chat .toggle-content .body {
-	color: #999;
+	color: #444;
```

Comment:
Same comment than on `self`, it's a pretty important content, but it's too dark as is. I'd go with `#666`, which passes AA levels.